src/navs/index.js

```diff
@@ -2033,16 +2033,6 @@ export const docsMenu = {
                 {
                     name: 'Tools',
                 },
-                {
-                    name: 'LLM insights',
-                    url: '/docs/product-analytics/llms',
-                    icon: 'IconMagicWand',
-                    color: 'yellow',
-                    badge: {
-                        title: 'Beta',
-                        className: 'uppercase !bg-blue/10 !text-blue !dark:text-white !dark:bg-blue/50',
-                    },
-                },
```

Comment:
Can we leave this in and have it direct to `/docs/ai-engineering/llm-insights` ?

It a good way to discover LLM insights, since the product tab is hidden after the scroll